### PR TITLE
Enable the use of HTML within subtitle

### PR DESCRIPTION
**Pull request by michael0liver**
Enables the use of HTML within subtitle.

I wanted to use `<b></b>` and `<br>` inside subtitle but it wasn't working when `typeit = false`. This change make sit possible :)

### CODE REVIEW

File: exampleSite/config.toml [hunk 1]
```diff
@@ -140,7 +140,7 @@ ignoreErrors = ["error-remote-getjson"]
           avatarURL = "/images/avatar.png"
           # title shown in home page (HTML format is supported)
           title = ""
-          # subtitle shown in home page
+          # subtitle shown in home page (HTML format is supported)
```
Comment (victor-pogor) [inline exampleSite/config.toml:143]: Can you add please `{{< version 0.1.1 changed >}}` to `config.toml` and `theme-documentation-basics/index.<LANG>.md` files?

Please take a look at [this PR](https://github.com/sunt-programator/CodeIT/pull/45/files#diff-1b323c3b150b637e63390b60e374400b9b58ed00ceaa944e1a0d4f335900100c).

**Review by victor-pogor — COMMENTED** [review]
Hi @michael0liver 

Thank you for your contributions.

We need to keep clean the version history. I leave a comment on how to add a version changed label in the documentation and config files.

**Comment by michael0liver** [comment]
@victor-pogor I think I have done it now.

**Comment by victor-pogor** [comment]
@all-contributors please add @michael0liver to code

**Review by victor-pogor — APPROVED** [review]
Thanks :)

**Pull request merged**